### PR TITLE
NXP-24273: add information to workflow event payloads

### PR DESCRIPTION
**Pull request by nmpcunha**

### CODE REVIEW

File: elements/nuxeo-document-info-bar/nuxeo-document-info-bar.html [hunk 1]
```diff
@@ -224,7 +221,7 @@
         if (confirm(this.i18n('documentPage.abandon.workflow.confirm'))) {
           this.$.worfklow.path = '/workflow/' + e.model.workflow.id;
           this.$.worfklow.remove().then(function() {
-            this.fire('workflowAbandoned');
+            this.fire('workflowAbandoned', {workflow: e.model.workflow});
```
Comment (Gabez0r) [inline elements/nuxeo-document-info-bar/nuxeo-document-info-bar.html]: Use space inside brackets:
```suggestion
            this.fire('workflowAbandoned', { workflow: e.model.workflow });
```

File: elements/workflow/nuxeo-document-task-assignment-popup.html [hunk 2]
```diff
@@ -97,9 +97,9 @@ <h2>[[i18n(task.name)]]</h2>
           this.params['comment'] = this.comment;
           this.params[this.action === 'delegate' ? 'delegatedActors' : 'actors'] = this.actors;
 
-          this.$.taskAssignment.put().then(function() {
+          this.$.taskAssignment.put().then(function(task) {
             this.$.assignmentDialog.close();
-            this.fire('workflowTaskAssignment');
+            this.fire('workflowTaskAssignment', {task: task});
```
Comment (Gabez0r) [inline elements/workflow/nuxeo-document-task-assignment-popup.html]:
```suggestion
            this.fire('workflowTaskAssignment', { task: task });
```

File: elements/workflow/nuxeo-document-task.html [hunk 3]
```diff
@@ -207,8 +207,8 @@ <h3>[[i18n(task.name)]]</h3>
             id: this.$.layout.element.task.id,
             variables: this.$.layout.element.task.variables
           };
-          this.$.taskRequest.put().then(function() {
-            this.fire('workflowTaskProcessed');
+          this.$.taskRequest.put().then(function(task) {
+            this.fire('workflowTaskProcessed', {task: task});
```
Comment (Gabez0r) [inline elements/workflow/nuxeo-document-task.html]:
```suggestion
            this.fire('workflowTaskProcessed', { task: task });
```

**Comment by Gabez0r** [comment]
👍

**Comment by guirenard** [comment]
:+1:

**Pull request merged**